src/main/webapp/js/build-message-div.js

```diff
@@ -27,4 +27,61 @@ function buildMessageDiv(message){
   messageDiv.appendChild(bodyDiv);
 
   return messageDiv;
+}
+
+function buildMessageInTimeline(message, flip){
+  const imageDiv = document.createElement('div');
+  imageDiv.classList.add("timeline-image");
+
+  const image = document.createElement('img');
+  image.classList.add("rounded-circle");
+  image.classList.add("img-fluid");
+  image.src ="/img/products-01.jpg";
+  image.alt ="";
+  imageDiv.appendChild(image);
+
+  const timeDiv = document.createElement('div');
+  timeDiv.classList.add('timeline-heading');
+  timeText = document.createElement('h4');
+  timeText.appendChild(document.createTextNode(new Date(message.timestamp)));
+  timeDiv.appendChild(timeText);
+
+  nameText = document.createElement('h4');
+  nameText.classList.add('subheading');
+  nameText.appendChild(document.createTextNode(message.user));
+  timeDiv.appendChild(nameText);
+
+  const bodyDiv = document.createElement('div');
+  bodyDiv.classList.add('timeline-body');
+  bodyText = document.createElement('p');
+  bodyText.appendChild(document.createTextNode(message.text));
```

Comment:
`createTextNode()` may remove the formatting from the markdown editor. Formatting with the new style may be tricky, the current way of displaying the message is with `innerHTML` as seen in `build-message-div.js`:  https://github.com/team-45-su-19/team-45-codeu/blob/0347e6e6247001da099afb19c10c3fc9d6564dc3/src/main/webapp/js/build-message-div.js#L28

This may or may not work with the new styles, may have to experiment a bit

Reply:
Now I added this simpleMDE. It can display the image, but one bug is that the image is not rescaled, so if there's a super big image, the image will be displayed in full-size. We can raise it as a issue to fix in the future